dm/config/security.go

```diff
@@ -18,5 +18,17 @@ type Security struct {
 	SSLCA         string   `toml:"ssl-ca" json:"ssl-ca" yaml:"ssl-ca"`
 	SSLCert       string   `toml:"ssl-cert" json:"ssl-cert" yaml:"ssl-cert"`
 	SSLKey        string   `toml:"ssl-key" json:"ssl-key" yaml:"ssl-key"`
-	CertAllowedCN []string `toml:"cert-allowed-cn" json:"cert-allowed-cn" yaml:"cert-allowed-cn"`
+	CertAllowedCN strArray `toml:"cert-allowed-cn" json:"cert-allowed-cn" yaml:"cert-allowed-cn"`
+}
+
+// used for parse string slice in flag
+type strArray []string
+
+func (i *strArray) String() string {
+	return "string array"
```

Comment:
How about returning a meaningful string, may as https://github.com/golang/go/blob/11f92e9dae96939c2d784ae963fa7763c300660b/src/flag/flag_test.go#L235

Reply:
update in 3e656ee